nautobot/dcim/models/devices.py

```diff
@@ -1935,3 +1943,20 @@ def clean(self):
                     raise ValidationError(
                         {f"{field}": f"The specified IP address ({ip}) is not assigned to this Virtual Device Context."}
                     )
+
+
+@extras_features("graphql")
```

Comment:
Any specific reasoning for omitting the other standard extras_features? I'd think webhooks and custom validators might be desirable, for example.